### DISCUSSION TITLE
Hello Party People

### DISCUSSION THREAD

**Discussion by Codeinator-0711** [Team-Earth-Quake-Detector/team_1_earth_quake_detector#2] [General]
I tried some things in GitHub. There´s much to do for us in this project. Need to get used to work with git....

**Comment by karinahasler** [comment]
I'm sure we can do it! #pythonexperts

**Reply by aliciamarleen** [reply]
This is gonna be so much fun! Looking forward to master the challenge with you #pythonexperts
